src/scheduler.rs

```diff
@@ -178,10 +193,17 @@ async fn fetch_idle(ctx: &Context, connection: &mut Imap, folder: Config) -> Int
                 connection.fake_idle(ctx, Some(watch_folder)).await
             }
         }
-        None => {
+        Ok(None) => {
             warn!(ctx, "Can not watch {} folder, not set", folder);
             connection.fake_idle(ctx, None).await
         }
+        Err(err) => {
+            warn!(
+                ctx,
+                "Can not watch {} folder, failed to retrieve config: {:?}", folder, err
```

Comment:
This message and message above probably should say `fetch` instead of `watch`, not really important.